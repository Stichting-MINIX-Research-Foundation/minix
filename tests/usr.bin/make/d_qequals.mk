# $NetBSD: d_qequals.mk,v 1.1 2012/03/17 16:33:14 jruoho Exp $

M= i386
V.i386= OK
V.$M ?= bug

all:
	@echo 'V.$M ?= ${V.$M}'
