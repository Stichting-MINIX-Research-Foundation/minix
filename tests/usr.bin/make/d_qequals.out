V.i386 ?= OK
